workflows/imagery/tests.yaml

```
@@ -0,0 +1,23 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: topo-imagery-test-script-
+spec:
+  serviceAccountName: workflow-runner-sa
+  podGC:
+    strategy: OnPodCompletion # Delete pod once its finished
+  nodeSelector:
+    karpenter.sh/capacity-type: "spot"
+  entrypoint: test-script
+  templates:
+  - name: test-script
+    script:
+      image: ghcr.io/linz/topo-imagery:v0.2.0-10-g34c210a
```

```suggestion
      image: ghcr.io/linz/topo-imagery:v0.2.0-11-g8411779
```

Is there a way to get the latest release without having to specify the version?

---

If we had a `latest` tag. Might be a good idea to have it for this workflow...